storage/innobase/xtrabackup/utils/debian/percona-xtrabackup-test-24.postinst

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+set -e
+case "$1" in
+  configure)
+    ln -s /opt/percona-xtrabackup/2.4/percona-xtranackup-test-24 /usr/share/percona-xtranackup-test-24
```

Comment:
what about upgrade if link exist already? -e and non-zero exit from ln will cause script failure.
also there is a mistype - xtranackup 2 times instead of xtrabackup.
third - -24 suffix is not needed because package is installed to separate versioned catalog, only symlink should be versioned in /usr/share